### PR TITLE
Move TimerInterface one level up

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

**Comment by WyriHaximus** [comment]
@clue you mentioned documentation needing updates. What did you have in mind?

File: src/LoopInterface.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace React\EventLoop;
 
-use React\EventLoop\Timer\TimerInterface;
+use React\EventLoop\TimerInterface;
```
Comment (clue) [inline src/LoopInterface.php]: May as well be removed now :+1:

Comment (WyriHaximus) [reply src/LoopInterface.php]: Fixed :+1:

**Comment by WyriHaximus** [comment]
Will rebase and squash once approved

**Pull request merged**